selfdrive/car/torque_data/override.toml

```diff
@@ -67,6 +67,7 @@ legend = ["LAT_ACCEL_FACTOR", "MAX_LAT_ACCEL_MEASURED", "FRICTION"]
 "HYUNDAI CUSTIN 1ST GEN" = [2.5, 2.5, 0.1]
 "LEXUS GS F 2016" = [2.5, 2.5, 0.08]
 "HYUNDAI STARIA 4TH GEN" = [1.8, 2.0, 0.15]
+"KIA EV9 2023" = [1.75, 1.75, 0.15]
```

Comment:
How did you arrive at these values?

Reply:
A good starting guess, drive around for a bit and update from the `liveTorqueParams` service in the logs